tests/components/light/test_init.py

```diff
@@ -453,7 +488,10 @@ def _mock_open(path, *args, **kwargs):
     """Test that light context works."""
     platform = getattr(hass.components, "test.light")
     platform.init()
-    assert await async_setup_component(hass, "light", {"light": {"platform": "test"}})
+    assert await async_setup_component(hass, "light",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)